src/by/naumenka/lesson21/Main.java

```diff
@@ -0,0 +1,45 @@
+package by.naumenka.lesson21;
+
+//Max,Winston,male,14
+//Jaime,Blossom,female,34
+//Darya,Naumenka,female,19
+
+import java.io.*;
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        ArrayList<Person> people = new ArrayList<>();
+        int ageCount, maleCount, femaleCount;
+        String separator = ",";
+
+        System.out.println("Введите ваш путь к файлу");
+        String path = new Scanner(System.in).nextLine(); //  src\\document.txt
+        BufferedReader br = new BufferedReader(new FileReader(path));
+
+        String document;
+        String[] infPerson;
+        while ((document = br.readLine()) != null) {
+            infPerson = document.split(separator);
+            people.add(new Person(infPerson[0], infPerson[1], infPerson[2], Integer.parseInt(infPerson[3].trim())));
+            System.out.println(document);
+        }
+
+        people.sort(Comparator.comparing(p -> (p.getFirstName() + p.getLastName())));
```

Comment:
According to the task, it should be second level sorting too )

Reply:
I'm a little confused as to what I did wrong with the sorting